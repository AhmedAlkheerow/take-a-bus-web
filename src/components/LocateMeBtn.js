import React from 'react';
import { GeolocateControl } from 'react-map-gl';
export default function LocateMeBtn() {
  return (
    <div className="btn btn-blue w-16 h-16 bg-blue-500 rounded-full flex flex-col justify-center items-center">
      <GeolocateControl
        positionOptions={{ enableHighAccuracy: true }}
        trackUserLocation={true}
      />
    </div>
  );
}
