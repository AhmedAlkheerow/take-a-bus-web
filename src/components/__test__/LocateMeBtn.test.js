import React from 'react';
import LocateMeBtn from '../LocateMeBtn';
import renderer from 'react-test-renderer';

// Snapshot test
test('Locate me button is renderer', () => {
  const tree = renderer.create(<LocateMeBtn />).toJSON();
  expect(tree).toMatchSnapshot();
});
