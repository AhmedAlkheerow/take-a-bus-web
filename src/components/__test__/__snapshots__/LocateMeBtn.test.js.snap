// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Locate me button is renderer 1`] = `
<div
  className="btn btn-blue w-16 h-16 bg-blue-500 rounded-full flex flex-col justify-center items-center"
/>
`;
